src/components/Statistic.js

```diff
@@ -11,7 +11,9 @@ class Statistic extends React.Component {
             return	<div className={styles.root}>
             <div className={styles.root2}>
```

Comment:
What does mean "root2"? Is it possible to change this class name to more informative?

Reply:
+1